### PR TITLE
Move sidenav arrow from right to left

### PR DESCRIPTION
**Pull request by FahadAminShovon**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR [adds/removes/fixes/replaces] this [feature/bug/etc]. 

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by angiejones** [comment]
Hi, can you provide info on what was changed and why? I don't see any difference in how the chevron is displayed compared to the existing site

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, {type ComponentProps, useEffect, useMemo} from 'react';
+import React, { type ComponentProps, useEffect, useMemo } from 'react';
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:1]:
```suggestion
import React, {type ComponentProps, useEffect, useMemo} from 'react';
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 2]
```diff
@@ -7,17 +7,18 @@ import {
   Collapsible,
   useCollapsible,
 } from '@docusaurus/theme-common';
-import {isSamePath} from '@docusaurus/theme-common/internal';
+import { isSamePath } from '@docusaurus/theme-common/internal';
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:10]:
```suggestion
import {isSamePath} from '@docusaurus/theme-common/internal';
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 3]
```diff
@@ -7,17 +7,18 @@ import {
...
 import {
   isActiveSidebarItem,
   findFirstSidebarItemLink,
   useDocSidebarItemsExpandedState,
 } from '@docusaurus/plugin-content-docs/client';
 import Link from '@docusaurus/Link';
-import {translate} from '@docusaurus/Translate';
+import { translate } from '@docusaurus/Translate';
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:17]:
```suggestion
import {translate} from '@docusaurus/Translate';
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 4]
```diff
@@ -7,17 +7,18 @@ import {
...
 import useIsBrowser from '@docusaurus/useIsBrowser';
 import DocSidebarItems from '@theme/DocSidebarItems';
-import type {Props} from '@theme/DocSidebarItem/Category';
+import type { Props } from '@theme/DocSidebarItem/Category';
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:20]:
```suggestion
import type {Props} from '@theme/DocSidebarItem/Category';
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 5]
```diff
@@ -48,7 +49,7 @@ function useAutoExpandActiveCategory({
  * see https://github.com/facebook/docusaurus/issues/3030
  */
 function useCategoryHrefWithSSRFallback(
-  item: Props['item'],
+  item: Props['item']
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:52]:
```suggestion
  item: Props['item'],
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 6]
```diff
@@ -83,20 +84,20 @@ function CollapseButton({
                 message: "Expand sidebar category '{label}'",
                 description: 'The ARIA label to expand the sidebar category',
               },
-              {label: categoryLabel},
+              { label: categoryLabel }
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:87]:
```suggestion
              {label: categoryLabel},
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 7]
```diff
@@ -83,20 +84,20 @@ function CollapseButton({
...
             )
           : translate(
               {
                 id: 'theme.DocSidebarItem.collapseCategoryAriaLabel',
                 message: "Collapse sidebar category '{label}'",
                 description: 'The ARIA label to collapse the sidebar category',
               },
-              {label: categoryLabel},
+              { label: categoryLabel }
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:95]:
```suggestion
              {label: categoryLabel},
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 8]
```diff
@@ -83,20 +84,20 @@ function CollapseButton({
...
             )
       }
       aria-expanded={!collapsed}
-      type="button"
-      className="clean-btn menu__caret"
+      type='button'
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:99]:
```suggestion
      type="button"
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 9]
```diff
@@ -83,20 +84,20 @@ function CollapseButton({
...
+      className='clean-btn menu__caret'
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:100]:
```suggestion
      className="clean-btn menu__caret"
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 10]
```diff
@@ -110,18 +111,18 @@ export default function DocSidebarItemCategory({
   index,
   ...props
 }: Props): JSX.Element {
-  const {items, label, collapsible, className, href} = item;
+  const { items, label, collapsible, className, href } = item;
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:114]:
```suggestion
  const {items, label, collapsible, className, href} = item;
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 11]
```diff
@@ -110,18 +111,18 @@ export default function DocSidebarItemCategory({
...
   const {
     docs: {
-      sidebar: {autoCollapseCategories},
+      sidebar: { autoCollapseCategories },
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:117]:
```suggestion
      sidebar: {autoCollapseCategories},
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 12]
```diff
@@ -110,18 +111,18 @@ export default function DocSidebarItemCategory({
...
     },
   } = useThemeConfig();
   const hrefWithSSRFallback = useCategoryHrefWithSSRFallback(item);
 
   const isActive = isActiveSidebarItem(item, activePath);
   const isCurrentPage = isSamePath(href, activePath);
 
-  const {collapsed, setCollapsed} = useCollapsible({
+  const { collapsed, setCollapsed } = useCollapsible({
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:125]:
```suggestion
  const {collapsed, setCollapsed} = useCollapsible({
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 13]
```diff
@@ -132,13 +133,13 @@ export default function DocSidebarItemCategory({
     },
   });
 
-  const {expandedItem, setExpandedItem} = useDocSidebarItemsExpandedState();
+  const { expandedItem, setExpandedItem } = useDocSidebarItemsExpandedState();
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:136]:
```suggestion
  const {expandedItem, setExpandedItem} = useDocSidebarItemsExpandedState();
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 14]
```diff
@@ -132,13 +133,13 @@ export default function DocSidebarItemCategory({
...
   // Use this instead of `setCollapsed`, because it is also reactive
   const updateCollapsed = (toCollapsed: boolean = !collapsed) => {
     setExpandedItem(toCollapsed ? null : index);
     setCollapsed(toCollapsed);
   };
-  useAutoExpandActiveCategory({isActive, collapsed, updateCollapsed});
+  useAutoExpandActiveCategory({ isActive, collapsed, updateCollapsed });
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:142]:
```suggestion
  useAutoExpandActiveCategory({isActive, collapsed, updateCollapsed});
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 15]
```diff
@@ -159,16 +160,18 @@ export default function DocSidebarItemCategory({
         {
           'menu__list-item--collapsed': collapsed,
         },
-        className,
-      )}>
+        className
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:163]:
```suggestion
        className,
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 16]
```diff
@@ -159,16 +160,18 @@ export default function DocSidebarItemCategory({
...
+      )}
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:164]:
```suggestion
      )}>
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 17]
```diff
@@ -159,16 +160,18 @@ export default function DocSidebarItemCategory({
...
+    >
       <div
         className={clsx('menu__list-item-collapsible', {
           'menu__list-item-collapsible--active': isCurrentPage,
-        })}>
+        })}
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:169]:
```suggestion
        })}>
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 18]
```diff
@@ -159,16 +160,18 @@ export default function DocSidebarItemCategory({
...
+      >
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:170]:
```suggestion
      
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 19]
```diff
@@ -190,8 +193,20 @@ export default function DocSidebarItemCategory({
           role={collapsible && !href ? 'button' : undefined}
           aria-expanded={collapsible && !href ? !collapsed : undefined}
           href={collapsible ? hrefWithSSRFallback ?? '#' : hrefWithSSRFallback}
-          {...props}>
-          {label}
+          {...props}
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:196]:
```suggestion
          {...props}>
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 20]
```diff
@@ -190,8 +193,20 @@ export default function DocSidebarItemCategory({
...
+        >
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:197]:
```suggestion
        
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 21]
```diff
@@ -205,7 +220,7 @@ export default function DocSidebarItemCategory({
         )}
       </div>
 
-      <Collapsible lazy as="ul" className="menu__list" collapsed={collapsed}>
+      <Collapsible lazy as='ul' className='menu__list' collapsed={collapsed}>
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:223]:
```suggestion
      <Collapsible lazy as="ul" className="menu__list" collapsed={collapsed}>
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 22]
```diff
@@ -161,14 +162,16 @@ export default function DocSidebarItemCategory({
         },
         className,
       )}>
+    
       <div
         className={clsx('menu__list-item-collapsible', {
           'menu__list-item-collapsible--active': isCurrentPage,
         })}>
+
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:170]:
```suggestion
```

File: site-new/src/theme/DocSidebarItem/Category/index.tsx [hunk 23]
```diff
@@ -161,14 +162,16 @@ export default function DocSidebarItemCategory({
         },
         className,
       )}>
+
```
Comment (angiejones) [inline site-new/src/theme/DocSidebarItem/Category/index.tsx:165]:
```suggestion

```

File: site-new/src/css/custom.css [hunk 24]
```diff
@@ -71,6 +71,8 @@
   --ifm-color-secondary: #9A1AFF;
   --ifm-background-color: #000;
   --ifm-font-color-base: #fff;
+
```
Comment (angiejones) [inline site-new/src/css/custom.css:74]:
```suggestion
```

**Comment by FahadAminShovon** [comment]
> Hi, can you provide info on what was changed and why? I don't see any difference in how the chevron is displayed compared to the existing site

sure,
Arrow orientation has changed from right to left

CleanShot 2024-09-04 at 14 48 25@2x

CleanShot 2024-09-04 at 14 47 18@2x

**Review by dayhaysoos — APPROVED** [review]
🚀

**Pull request merged**